docs/outputs.rst

```diff
@@ -22,207 +21,330 @@ FMRIPrep generates three broad classes of outcomes:
      the same-subject's T1w space or into MNI space.
 
   3. **Additional data for subsequent analysis**, for instance the transformations
-     between different spaces or the estimated confounds.
+     between different spaces or the estimated confounds_.
 
 
-fMRIPrep outputs conform to the :abbr:`BIDS (brain imaging data structure)`
+*fMRIPrep* outputs conform to the :abbr:`BIDS (brain imaging data structure)`
 Derivatives specification (see `BIDS Derivatives RC1`_).
 
 Visual Reports
 --------------
-
-FMRIPrep outputs summary reports, written to ``<output dir>/fmriprep/sub-<subject_label>.html``.
+*fMRIPrep* outputs summary reports, written to ``<output dir>/fmriprep/sub-<subject_label>.html``.
 These reports provide a quick way to make visual inspection of the results easy.
 Each report is self contained and thus can be easily shared with collaborators (for example via email).
 `View a sample report. <_static/sample_report.html>`_
 
-
-Preprocessed data (fMRIPrep *derivatives*)
-------------------------------------------
-
+Derivatives of *fMRIPrep* (preprocessed data)
+---------------------------------------------
 Preprocessed, or derivative, data are written to
 ``<output dir>/fmriprep/sub-<subject_label>/``.
 The `BIDS Derivatives RC1`_ specification describes the naming and metadata conventions we follow.
 
 Anatomical derivatives are placed in each subject's ``anat`` subfolder:
+::
 
-- ``anat/sub-<subject_label>_[space-<space_label>_]desc-preproc_T1w.nii.gz``
-- ``anat/sub-<subject_label>_[space-<space_label>_]desc-brain_mask.nii.gz``
-- ``anat/sub-<subject_label>_[space-<space_label>_]dseg.nii.gz``
-- ``anat/sub-<subject_label>_[space-<space_label>_]label-CSF_probseg.nii.gz``
-- ``anat/sub-<subject_label>_[space-<space_label>_]label-GM_probseg.nii.gz``
-- ``anat/sub-<subject_label>_[space-<space_label>_]label-WM_probseg.nii.gz``
-
-Template-normalized derivatives use the space label ``MNI152NLin2009cAsym``, while derivatives in
+  sub-<subject_label>/
+    anat/
+      sub-<subject_label>[_space-<space_label>]_desc-preproc_T1w.nii.gz
+      sub-<subject_label>[_space-<space_label>]_desc-brain_mask.nii.gz
+      sub-<subject_label>[_space-<space_label>]_dseg.nii.gz
+      sub-<subject_label>[_space-<space_label>]_label-CSF_probseg.nii.gz
+      sub-<subject_label>[_space-<space_label>]_label-GM_probseg.nii.gz
+      sub-<subject_label>[_space-<space_label>]_label-WM_probseg.nii.gz
+
+Spatially-standardized derivatives are denoted with a space label,
+such as ``MNI152NLin2009cAsym``, while derivatives in
 the original ``T1w`` space omit the ``space-`` keyword.
 
 Additionally, the following transforms are saved:
```

Comment:
Double-colons follow the [rules](https://documentation.help/Sphinx/rest.html#source-code):

> * If it occurs as a paragraph of its own, that paragraph is completely left out of the document.
> * If it is preceded by whitespace, the marker is removed.
> * If it is preceded by non-whitespace, the marker is replaced by a single colon.


```suggestion
Additionally, the following transforms are saved::
```